### PR TITLE
[ch4294] Add autocomplete address via google maps api

### PR DESCRIPTION
**Pull request by cacahouwete**
https://app.clubhouse.io/enmarche/story/4194/int%C3%A9grer-le-place-autocomplete-au-formulaire-de-don

### CODE REVIEW

File: templates/membership/join.html.twig [hunk 1]
```diff
@@ -26,8 +27,65 @@
...
+                        domElement.value += ' ';
+                    }
+                    domElement.value += val;
+                    domElement.dispatchEvent(new InputEvent('input'));
+                }
+            }
+        }
     </script>
     <script src="https://www.google.com/recaptcha/api.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key={{ google_maps_api_key }}&libraries=places&callback=initAutocomplete"
```
Comment (hhamon) [inline templates/membership/join.html.twig]: `&` should be `&amp;`

File: templates/membership/join.html.twig [hunk 2]
```diff
@@ -11,6 +11,7 @@
 {% block open_graph_image_height 630 %}
 
 {% block stylesheets %}
+    <link type="text/css" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
```
Comment (B-Galati) [inline templates/membership/join.html.twig]: I guess we don't want to use public CDN for this case no ?

File: templates/membership/join.html.twig [hunk 3]
```diff
@@ -22,12 +23,19 @@
...
+                /** @type {!HTMLInputElement} */(document.getElementById('autocomplete')),
+                {types: ['geocode']});
+            Kernel.onLoad(function() {
+                App.runJoin('{{ form.vars.form_type_class|e('js') }}', autocomplete);
+                {{ js.address(form.address) }}
+            });
+        }
     </script>
     <script src="https://www.google.com/recaptcha/api.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key={{ google_maps_api_key }}&amp;libraries=places&amp;callback=initAutocomplete"
```
Comment (B-Galati) [inline templates/membership/join.html.twig]: It looks like a security issue.
We should not exposed our secret in our sourcecode.
I saw that we did the same in many places in the code. Someone have an explanation ?
@ottaviano @illusionOfParadise @thomasbisignani @Remg  ?

File: templates/membership/join.html.twig (hunk 1 above)
Comment (ottaviano) [reply templates/membership/join.html.twig]: fixed, thanks @hhamon 👍

File: templates/membership/join.html.twig (hunk 2 above)
Comment (ottaviano) [reply templates/membership/join.html.twig]: removed :D

File: templates/membership/join.html.twig (hunk 3 above)
Comment (ottaviano) [reply templates/membership/join.html.twig]: @B-Galati, to secure the API key @atamano should restrict access for this key on GCloud console if not done yet

**Pull request merged**